### DISCUSSION TITLE
can we customize the Web3NetworkSwitch button or how can we get the  image or details about connected Network

### DISCUSSION THREAD

**Discussion by can2112** [WalletConnect/walletconnect-monorepo#2632] [Developer Support]
I was facing a issue about, Web3NetworkSwitch button doesn't give us suport to customize, is there any way we can customize the button or we can get the logo of selected netwrok chain, there is one wagmi hook which give lots of info but not the logo or any kind of image of the selected network.

**Comment by glitch-txs** [comment]
The selected network should be the current chain.
You can create a custom button with the useWeb3Modal hook, using the `open` function you can pass `SelectNetwork` as argument
https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/hooks#useweb3modal

**Reply by KardoseR** [reply]
How do I pass in SelectNetwork as an argument?

<button onClick={open("SelectNetwork")}>Open</button>
